documentation/asciidoc/accessories/camera/camera_installing.adoc

```diff
@@ -14,41 +14,45 @@ Depending on the model, the camera may come with a small piece of translucent bl
 
 === Enabling the Camera
 
-==== Using the desktop
-
-Select `Preferences` and `Raspberry Pi Configuration` from the desktop menu: a window will appear. Select the `Interfaces` tab, then click on the `enable camera` option. Click `OK`. You will need to reboot for the changes to take effect.
-
-==== Using the command line
-
-Open the `raspi-config` tool from the terminal:
+Before proceeding, we recommend ensuring that your kernel, GPU firmware and applications are all up to date. You'll need an internet connection for this to work correctly.
 
 [,bash]
 ----
-sudo raspi-config
+sudo apt update
+sudo apt full-upgrade
 ----
 
-Select `Interfacing Options` then `Camera` and press `Enter`. Choose `Yes` then `Ok`. Go to `Finish` and you'll be prompted to reboot.
+==== Using `libcamera`
 
-=== Setting up the Camera Software
+Raspberry Pi's three official cameras (V1, V2 and HQ camera) will be detected and enabled automatically when using a Raspberry Pi OS based on _Bullseye_ or later.
```

Comment:
I wonder if we ought to delay merging this until after we've released a Raspberry Pi OS Bullseye image, to avoid any user-confusion?

Reply:
I did check with Gordon and he was OK with it... after all it's no secret that Bullseye is coming and trailing some of its new features isn't a bad thing?

Reply:
We've already had a couple of emails and forum posts bemoaning the documentation doesn't tell them how to update to Bullseye. I might put a holding paragraph into the update & upgrade section tell folks that Bullseye isn't out yet. This would certainly need a `[NOTE]` block saying that Bullseye isn't out and to watch the blog (or something like that).

Reply:
We've not on the whole published official instructions on updating from one major version to another because we cannot be sure it will work in all cases. When we have said something its with LOTS of try this at your own peril stuff etc. recommendation has always been get a new SD card and a new installation.

Reply:
Yup. There are instructions somewhere but they have heavy warnings around them.